content/applications/websites/livechat/overview/get_started.rst

```diff
@@ -2,82 +2,200 @@
+
+.. image:: get_started/live-chat-enable-setting.png
    :align: center
-   :height: 300
    :alt: View of the settings page and the live chat feature for Odoo Live Chat
 
-Select the channel to be linked to your website or create one on the fly.
+A :guilabel:`Live Chat Channel` will be created by default and automatically selected in the
+drop-down.
+
+Create a new live chat channel
+==============================
 
-.. image:: get_started/channels.png
+To create a new :guilabel:`Live Chat Channel`, go to :menuselection:`Dashboard --> Live Chat -->
+New`. Enter the name of the new channel in the :guilabel:`Channel Name` field.
+
+.. image:: get_started/live-chat-open-channel.png
    :align: center
-   :alt: View of a live chat channel form for Odoo Live Chat
-
-| For both scenarios, under:
-| - **Operators**: add agents to respond to the chat requests. Add as many as you like, and keep in
-  mind that operators that do not show any activity in Odoo for more than 30min are considered
-  disconnected.
-| - **Options**: set the default text to be shown on the live chat button; an automated welcome
-  message to be seen by visitors when a conversation is initiated, and the text that prompts the
-  user to initiate a chat.
-| - **Channel Rules**: choose an action for a given URL, and/or per country. In the example below,
-  the chat window automatically pops-up 3 seconds after users (from any country) land on the
-  contact us page.
-
-.. image:: get_started/rules.png
+   :alt: View of a live chat channel form for Odoo Live Chat.
+
+To configure the remaining tabs, follow the steps below.
+
+Operators
+---------
+
+*Operators* are the users who will respond to live chat requests from customers. The user who
+originally created the live chat channel will be added as an operator by default.
+
+To add additional users, navigate and click on the :guilabel:`Live Chat Channel` from the
+:guilabel:`Website Live Chat Channels` dashboard, and on the :guilabel:`Operators` tab, click
+:guilabel:`ADD`.
+
+Then, click the check box next to the users to be added, and click :guilabel:`SELECT`.
+:guilabel:`New` operators can be created and added to the list, as well, by filling out the
+:guilabel:`Create Operators` form and then clicking :guilabel:`SAVE & CLOSE` (or :guilabel:`SAVE &
+NEW` for multiple record creations).
+
+As well, current operators can be edited or removed by clicking on their respective boxes in the
+:guilabel:`Operators` tab, and then adjusting their form values, or by using one of the form
+buttons located at the bottom of the form, such as :guilabel:`REMOVE`.
+
+Options
+-------
+
+The :guilabel:`Options` tab contains the visual and text settings for the live chat window.
+
+Change the text in the :guilabel:`Text of the Button` field to update the greeting displayed in the
+text bubble when the live chat button appears on the website.
+
+Edit the :guilabel:`Welcome Message` to change the message a visitor sees when they open the chat
```

Comment:
I agree. I wanted to keep the color selections at the end (because they required additional information) but it does make sense to move the button information up for the doc flow.